mobile/src/context/BookContext.tsx

```diff
@@ -1,191 +1,57 @@
-import React, { createContext, useReducer } from 'react';
-
+import React, { createContext, useReducer, useContext } from 'react';
+import { APIContext } from './APIContext';
 import { Book } from '../models/Book';
 
 type BookState = {
     books: Book[], // holds all the books we have stored
     fetchBooks: () => void, // api call to get books
+    loading: boolean, // lets us know if the books are being loaded or not
 };
 
 const initialState: BookState = {
   books: [],
   fetchBooks: () => {},
+  loading: true,
```

Comment:
Technically the context is _not_ loading anything until the first call to fetchBooks(), so this should be false.